Week-4/4.1/CustomPromise.js

```diff
@@ -0,0 +1,127 @@
+const STATE = {
```

Comment:
Consider using Enum

Reply:
enums can be used only with typescript. In my case, I have implemented an enum by create an object.

Reply:
https://www.sohamkamani.com/javascript/enums/